### ISSUE TITLE
Force phpunit to run all tests and do not auto-skip tests when any error occurs

### ISSUE THREAD

**Issue by mjafari98** [sebastianbergmann/phpunit#4749]
When you run the tests in a project containing a syntax error, using PHPUnit, it will run the first test and skips the remaining tests.

I think this behaviour is not normal; Because its output looks like this:
```
Tests: 9, Assertions: 0, Errors: 1, Skipped: 8.
```

The expected behaviour is this:
```
Tests: 9, Assertions: 0, Errors: 9, Skipped: 0.
```

I think, having a `--disable-auto-skip` option would be nice to have or something similar.

Regards.

**Comment by sebastianbergmann** [comment]
What you describe is how PHPUnit should already behave.

Please provide a **minimal**, **self-contained**, reproducing test case that shows the problem you are reporting.

Without such a minimal, self-contained, reproducing test case I will not be able to investigate this issue.

**Comment by mjafari98** [comment]
> What you describe is how PHPUnit should already behave.
> 
> Please provide a **minimal**, **self-contained**, reproducing test case that shows the problem you are reporting.
> 
> Without such a minimal, self-contained, reproducing test case I will not be able to investigate this issue.

Having a file named `isPrime.php` in a tmp directory; And a `test` folder in the same directory; And a test file in `test/IsPrimeTest.php`, you will face the error running `php phpunit-9.5.7.phar .` command.

Contents of files:

`isPrime.php`:
```php
<?php

function isPrime($n)
{
    for ($i = 2; $i <= $n / 2 = 1; $i++) {
        if ($n % $i == 0) {
            return false;
        }
    }
    return true;
}
```

`test/IsPrimeTest.php`:
```php
<?php

use \PHPUnit\Framework\TestCase;

final class IsPrimeTest extends TestCase
{
    public static $answer_file_path = __DIR__.'/../isPrime.php';

    public static function setUpBeforeClass(): void
    {
        require_once self::$answer_file_path;
    }

    public function testIsPrimeZero()
    {
        $this->assertFalse(isPrime(0));
    }

    public function testIsPrimeOne()
    {
        $this->assertFalse(isPrime(1));
    }
}
```


Output:
```txt
PHPUnit 9.5.7 by Sebastian Bergmann and contributors.

ES                                                                  2 / 2 (100%)

Time: 00:00, Memory: 18.00 MB

There was 1 error:

1) IsPrimeTest::testIsPrimeZero
ParseError: syntax error, unexpected token "=", expecting ";" in /home/isprime/isPrime.php:5
Stack trace:
#0 phar:///home/isprime/phpunit-9.5.7.phar/phpunit/Framework/TestSuite.php(468): IsPrimeTest::setUpBeforeClass()
#1 phar:///home/isprime/phpunit-9.5.7.phar/phpunit/Framework/TestSuite.php(508): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#2 phar:///home/isprime/phpunit-9.5.7.phar/phpunit/TextUI/TestRunner.php(489): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#3 phar:///home/isprime/phpunit-9.5.7.phar/phpunit/TextUI/Command.php(124): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#4 phar:///home/isprime/phpunit-9.5.7.phar/phpunit/TextUI/Command.php(93): PHPUnit\TextUI\Command->run(Array, true)
#5 /home/isprime/phpunit-9.5.7.phar(1156): PHPUnit\TextUI\Command::main()
#6 {main}
ERRORS!
Tests: 2, Assertions: 0, Errors: 1, Skipped: 1.

```

**Comment by sebastianbergmann** [comment]
Both your tests operate on the same file that contains a syntax error. And as you require that file in a `setUpBeforeClass()` method, the exception (for the parse error) raised in that method skips the execution of subsequent tests of that test case class.

Here is a different example:

#### `Test.php`
```php
<?php declare(strict_types=1);
use PHPUnit\Framework\TestCase;

final class Test extends TestCase
{
    public function testOne(): void
    {
        require __DIR__ . '/A.php';
    }
    
    public function testTwo(): void
    {
        require __DIR__ . '/B.php';

        $this->assertTrue((new B)->method());
    }
}
```
#### `A.php`
```php
<?php declare(strict_types=1);
cla ss A
{
}
```
#### `B.php`
```php
<?php declare(strict_types=1);
class B
{
    public function method(): bool
    {
        return true;
    }
}
```
```
$ php phpunit-9.5.phar Test.php 
PHPUnit 9.5.7 by Sebastian Bergmann and contributors.

E.                                                                  2 / 2 (100%)

Time: 00:00.001, Memory: 18.00 MB

There was 1 error:

1) Test::testOne
ParseError: syntax error, unexpected identifier "ss"

/home/sb/A.php:2

ERRORS!
Tests: 2, Assertions: 1, Errors: 1.
```
**TL;DR:** PHPUnit works correctly.
